.github/workflows/update-docs.yaml

```diff
@@ -0,0 +1,39 @@
+# this workflow is used to update the Regal content at docs.styra.com
+# when it changes in this repo.
+name: Update Docs
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  update-docs:
+    name: Update Docs
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Generate GitHub App Token
+        id: generate_token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.PLATFORM_AUTOMATION_GH_APP_ID }}
+          private_key: ${{ secrets.PLATFORM_AUTOMATION_GH_APP_PEM_KEY }}
+
+      - name: Trigger Update in docs repo
+        run: |
+          curl -X "POST" "https://api.github.com/repos/StyraInc/docs/actions/workflows/update-regal.yaml/dispatches" \
+            -H 'Accept: application/vnd.github+json' \
+            -H 'Authorization: Bearer ${{ steps.generate_token.outputs.token }}' \
+            -H 'X-Github-Api-Version: 2022-11-28' \
+            -H 'Content-Type: application/json; charset=utf-8' \
+            -d $'{
```

Comment:
What does the `$` do here? 🤔

Reply:
I actually think in this case it's not doing anything, but if there were `\n`s in there, they'd be handled as new lines. I think, just a bash thing as I understand it.

Reply:
I have removed it, it's not needed. It's just set in the generated commands from my REST gui thingy.

Reply:
Interesting! I'll have to read up on that.